build/media-suite_compile.sh

```diff
@@ -1227,7 +1227,7 @@ _check=(SvtHevcEnc.pc libSvtHevcEnc.dll.a svt-hevc/EbApi.h
 if [[ $svthevc = "y" ]] && do_vcs "https://github.com/OpenVisualCloud/SVT-HEVC.git"; then
     do_uninstall bin-video/SVT-Hevc/libSvtHevcEnc.dll.a "${_check[@]}"
     do_patch "https://github.com/OpenVisualCloud/SVT-HEVC/pull/213.patch"
-    do_patch "http://0x0.st/zQP4.patch"
+    do_patch "https://gist.githubusercontent.com/1480c1/a5fbbff6aa2cb9880275ae6ad63bd9df/raw/59922fb6a7b242627dea517acc6d21dae6b9aee2/0001-transition-between-y4m-and-cmakelists.patch"
```

Comment:
```suggestion
    do_patch "https://gist.githubusercontent.com/1480c1/a5fbbff6aa2cb9880275ae6ad63bd9df/raw/0001-transition-between-y4m-and-cmakelists.patch"
```
If you remove the commit path parameter it seems to point to the HEAD, it seems.